config/settings.yml

```diff
@@ -454,6 +454,10 @@ flipper:
   github_organization: department-of-veterans-affairs
   github_team: <%= ENV['flipper__github_team'] %>
   mute_logs: <%= ENV['flipper__mute_logs'] %>
+form0781_remediation:
+  aws:
+    bucket: <%= ENV['evidence_remediation__aws__bucket'] %>
```

Comment:
I checked AWS for this and it's not configured in the correct format. `env_vars` needs to be part of the path. See [this step of the docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/settings-and-parameter-store#SettingsandParameterStore-Step3:(Optional)AddthevaluetotheParamStore).

Reply:
Hi @rmtolmach, I'm taking over this work and moving it forward. I deleted the incorrectly pathed buckets and [created buckets for each env](https://us-gov-west-1.console.amazonaws-us-gov.com/systems-manager/parameters/?region=us-gov-west-1&tab=Table#list_parameter_filters=Name:Contains:evidence_remediation) with the correct path. How does this look now?
![image](https://github.com/user-attachments/assets/806d60f6-dfb1-4415-b728-57354d0700d0)